x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/filters/integrations_selector.tsx

```diff
@@ -91,7 +91,7 @@ export function IntegrationsSelector({
     <EuiFilterButton
       data-test-subj="datasetQualityIntegrationsSelectableButton"
       iconType="arrowDown"
-      badgeColor="success"
+      badgeColor="accentSecondary"
```

Comment:
We should keep using `success` here. Badges actually don't support `accentSecondary` and this value will be removed again soon. (same for any following changes of `badgeColor`)

Reply:
I actually thought that, as it's not really a success outcome of any type, `accentSecondary` was the right choice here based on the migration guidelines.

Also, seems the badges support the `accentSecondary` color:

<img width="714" alt="Screenshot 2024-12-12 at 12 07 59" src="https://github.com/user-attachments/assets/f10c43ad-a2e7-4cbd-9700-13c5665ab9fc" />

If you can confirm this should anyway be reverted to success I'll do it, just want to clarify why I switched it.

Reply:
Yes, they currently _do_ support this variant but latest design decisions actually now don't expect this to be supported, so [we're removing it again](https://github.com/elastic/eui/pull/8227) 😅 

Yes, please revert to success in this case 🙏

Reply:
Addressed with [style(logs): revert badges color to success](https://github.com/elastic/kibana/pull/202746/commits/169eee0b76b79465d4c5f1787d315bb323c29dbc)